cdbot/cogs/cyber.py

```diff
@@ -494,6 +448,94 @@ async def on_message(self, message: Message):
                 await message.channel.send(f"{message.author.mention}  |  {response}")
                 break
 
+    @Cog.listener()
+    async def on_ready(self):
+        readmeFile = "cdbot/data/readme.json"
+        with open(readmeFile, "rb") as f:
+            bytes = f.read()
+            readmeHash = hashlib.sha256(bytes).hexdigest()
+
+        fileHash = None
+        testChannel = await self.bot.fetch_channel(DEV_TESTING_CHANNEL_ID)
+        messages = await testChannel.history().flatten()
+        hashMsg = None
+        for msg in messages:
+            if msg.content[:6] == "/hash/" and msg.author == self.bot.user:
+                fileHash = msg.content[6:]
+                hashMsg = msg
+                break
+
+        if readmeHash == fileHash:
+            pass
+        else:
```

Comment:
Why not have `if readmeHash != fileHash`?

Reply:
idk ive just fixed it